src/View/Sidebar/BookmarkRow.vala

```diff
@@ -243,7 +243,7 @@ public class Sidebar.BookmarkRow : Gtk.ListBoxRow, SidebarItemInterface {
         }
     }
 
-    protected void popup_context_menu (Gdk.EventButton event) {
+    protected virtual void popup_context_menu (Gdk.EventButton event) {
```

Comment:
Why do we need this to be `virtual` now?

Reply:
It is overridden in `DriveRow.vala`